mmv1/products/iamworkforcepool/WorkforcePoolProvider.yaml

```
@@ -70,6 +70,14 @@ examples:
       org_id: :ORG_ID
     ignore_read_extra:
       - 'oidc.0.client_secret.0.value.0.plain_text'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'iam_workforce_pool_provider_oidc_upload_key'
```

We can probably leave this out of the examples in the provider docs since there's not significant additional complexity; we're just confirming the resource works.

```suggestion
    name: 'iam_workforce_pool_provider_oidc_upload_key'
    skip_docs: true
```